amplify/README.md

```diff
@@ -0,0 +1,8 @@
+# Getting Started with Amplify CLI
```

Comment:
I don't think me or Tristan got this README when we were using amplify, our versions may be outdated